### PR TITLE
Moving text in demo

### PR DESCRIPTION
**Pull request by damaskino**
This branch just adds the gradually displayed text to the demo so we can have our core features at a glance in the demo. 
The demo is still a little buggy (e.g., switching in and out of the demo produces weird text), but for now, I think it would be enough to just make sure that the added npc and player classes have not been broken, then we fix the bugs in the master.
Let me know what you think @3ter :)

### CODE REVIEW

File: internal/scene/scene.go [hunk 1]
```diff
@@ -55,23 +55,6 @@ func SetWindowForAllScenes(win *pixelgl.Window) {
 	window = win
 }
 
-// TODO: This has probably been copied here as a reference.
```
Comment (3ter) [inline internal/scene/scene.go:58]: Yeah, housekeeping :+1:

File: internal/scene/sceneDemo.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ func (s *Scene) InitDemoScene() {
 	var trackPath = "../assets/"
 	s.trackMap = make(map[int]*effects.Volume)
 	for index, element := range trackArray {
-		fmt.Println(index, trackPath, element)
+		//fmt.Println(index, trackPath, element)
```
Comment (3ter) [inline internal/scene/sceneDemo.go:33]: Did you leave that debug statement in there on purpose?

**Review by 3ter — COMMENTED** [review]
I just wanted to try out the review now that I looked it up how to do it. It's cool imho.

File: internal/scene/sceneDemo.go (hunk 2 above)
Comment (damaskino) [reply internal/scene/sceneDemo.go:33]: We can leave the whole loop as a comment, this was just way back when I needed to check if the tracks are actually getting added to the array. There is no need to have it printed out now.

File: internal/scene/scene.go (hunk 1 above)
Comment (damaskino) [reply internal/scene/scene.go:58]: Yup, I take it it's fine now, with the commented code removed, if I'm not mistaken, so resolving this conversation, too. :)

**Pull request merged**